### DISCUSSION TITLE
داشبورد روی https در کانفیگ های multi-port و single-port-proxy

### DISCUSSION THREAD

**Discussion by SaintShit** [Gozargah/Marzban#60] [Show and tell]
برای داشتن خود داشبورد روی https

اگر از کانفیگ fully single port استفاده میکنید، نیازی به تغییرات اضافی نیست و با فعال کردن tls روی کانفیگ xray، برای دامنه هم فعال میشه و داشبورد روی 
`https://YOUR_DOMAIN/dashboard`
میاد.
اگر از کانفیگ های دیگه مثل multi port یا single port proxy استفاده میکنید، باید توی فایل env دوتا متغیر قرار بدید و فایل های ssl رو بهش معرفی کنید به شکل زیر:

```bash
UVICORN_SSL_CERTFILE = "/var/lib/marzban/certs/fullchain.pem"
UVICORN_SSL_KEYFILE = "/var/lib/marzban/certs/key.pem"
```

بعد سرویستون رو ری استارت کنید.

حواستون باشه که پورت پیش فرض https 443 هست پس اگر میخواید داشبورد رو‌ دقیقا روی همون
`https://YOUR_DOMAIN/dashboard`

داشته باشید پورت اپلیکیشن رو تغییر بدید.

**Comment by rbsdotnet** [comment]
درود برای اینکه تمام کانفیگ ها روی پورت 443 باشه و پنل هم روی پورت 443 باشه باید از چه روشی این کار رو انجام بدم؟
ممنون میشم راهنمایی کنید
